tests/tacacs/tac_plus.conf.j2

```diff
@@ -10,30 +10,56 @@ accounting file = /var/log/tac_plus.acct
 key = {{ tacacs_passkey }}
 
 group = netadmin {
-	default service = permit
-	service = exec {
-		priv-lvl = 15
-	}
+    default service = permit
+    service = exec {
+        priv-lvl = 15
+    }
 }
 
 user = {{ tacacs_rw_user }} {
-	login = des {{ tacacs_rw_user_passwd }}
-	member = netadmin
+    login = des {{ tacacs_rw_user_passwd }}
+    member = netadmin
 }
 
 group = netuser {
-	default service = permit
-	service = exec {
-		priv-lvl = 1
-	}
+    default service = permit
+    service = exec {
+        priv-lvl = 1
+    }
 }
 
 user = {{ tacacs_ro_user }} {
-	login = des {{ tacacs_ro_user_passwd }}
-	member = netuser
+    login = des {{ tacacs_ro_user_passwd }}
+    member = netuser
 }
 
 user = {{ tacacs_jit_user }} {
-	login = des {{ tacacs_jit_user_passwd }}
-	member = {{ tacacs_jit_user_membership }}
+    login = des {{ tacacs_jit_user_passwd }}
+    member = {{ tacacs_jit_user_membership }}
+}
+
+user = {{ tacacs_authorization_user }} {
+    login = des {{ tacacs_authorization_user_passwd }}
+    member = netuser
+    cmd = /usr/bin/cat {
+        deny .*
+    }
+    # disable following command for UT test_bypass_authorization
+    cmd = /usr/local/bin/config {
+        deny   tacacs
+        permit .*
+    }
+    cmd = /usr/bin/python {
+        deny .*
+    }
+    cmd = /usr/bin/find {
+        deny   -exec
+        permit .*
+    }
+    cmd = /usr/lib/x86_64-linux-gnu/ld-version.so {
```

Comment:
>version [](http://example.com/codeflow?start=39&length=7)

I cannot find this file on a Debian 10. Is it expected?

Reply:
I see you use this name as a placeholder. How about `/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2` which is a existing file and make more sense?